docs/form-datasets.rst

```diff
@@ -1,32 +1,47 @@
+.. spelling:word-list::
+  aae
+  Ifedore
+  Magama
+  da
+  ee
+  Xue
+  Dia
+
 ************************
-Form Datasets
+Form datasets
 ************************
 
-ODK forms can use datasets in a variety of ways. These datasets can be either internal or external to the form.
+Workflows sometimes require access to historical data, standard data tables, lists of people, or other information. You can include that data directly in the form, attach it as a CSV file, or define it in an Entity List which can be shared between forms. All of these sources of data can be used to build select questions or to look up specific values.
 
 Internal datasets are defined in the **choices** sheet of an XLSForm and are typically used as choices for :ref:`selects <select-widgets>`. You can also define a dataset in the **choices** sheet to :ref:`look up values <referencing-values-in-datasets>` based on user input. Learn how to define internal datasets in the section on :ref:`selects <select-widgets>`.
 
-External datasets are useful when:
+External datasets can either be files that get attached to the form or :doc:`Entity Lists <central-entities>`. Entity Lists can be thought of as CSVs that are managed by ODK Central so that data collected by one form submission can be automatically accessible in other forms in the same project. The also can be shared between forms and can be updated by form submissions.
+
+External datasets, whether they are attached files or Entity Lists, are useful when:
 
-* data comes from another system. Using data files attached to the form generally requires fewer steps than adding the data to a form definition.
-* data changes frequently. One or more data file attached to the form can be updated without modifying the form definition.
-* data is reused between forms. It may be easier to attach the same data file to multiple forms instead of copying the data into all the form definitions.
-* the same forms are used in different contexts. For example, the exact same form definition could be used in multiple countries with different data files listing regions, local products, etc.
+* data comes from another system. Using external datasets generally requires fewer steps than adding the data to a form definition.
+* data needs to change over time. External datasets can be updated without modifying the form definition.
+* data is reused between forms. It may be easier to attach the same data file to multiple forms instead of copying the data into all the form definitions. Entity Lists can be shared between forms.
```

Comment:
Entity Lists are even easier because they be shared between forms and automatically kept up-to-date.